CI/bench/generate_amlb_user_dir.sh

```diff
@@ -4,9 +4,10 @@ REPOSITORY=$1
 BRANCH=$(basename $2)
 SHORT_SHA=$3
 PR_NUMBER=$4
+FOLDS=$5
 
 # generate tabular configs
-python $(dirname "$0")/tabular/generate_framework.py --repository https://github.com/$REPOSITORY.git --branch $BRANCH
+python $(dirname "$0")/tabular/generate_framework.py --repository https://github.com/$REPOSITORY.git --branch $BRANCH --folds_to_run $FOLDS
```

Comment:
Won't this crash if folds isn't specified?

Reply:
`--folds_to_run $FOLDS` should only exist in the command if `$FOLDS` is specified

Reply:
added more robust check